internal/builder/packages.go

```diff
@@ -157,6 +159,11 @@ func BuildPackage(options BuildOptions) (string, error) {
 		return "", errors.Wrap(err, "copying package contents failed")
 	}
 
```

Comment:
Please leave `logger.Debugf` here for consistency with other steps (clear target, encode dashboards, resolve fields).

Reply:
Here it does different things depending on the situation, so it will log two messages for the same thing, do we still want the log message here?

Reply:
Ok, message added, now it looks like this:

When copied from the repo:
```
2022/08/02 15:59:57 DEBUG Copy license file if needed
2022/08/02 15:59:57  INFO License text found in "/home/jaime/gocode/src/github.com/elastic/integrations/LICENSE.txt" will be included in package
```

When already available in the package:
```
2022/08/02 16:01:32 DEBUG Copy license file if needed
2022/08/02 16:01:32 DEBUG License file in the package will be used
```